src/redux/libraryIndex.ts

```diff
@@ -25,7 +25,8 @@ interface Library {
         address: {
             street: string, 
             city: string, 
-            state: string
+            state: string,
+            zip: number
```

Comment:
Good catch on the missing Zip code! That wasn't included in our mock server sample data.